.github/workflows/nigthly-release.yml

```diff
@@ -0,0 +1,49 @@
+name: Nightly Release
+
+on: 
+  schedule:
+    # This runs at 00:00 UTC every day
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  nightly:
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+    steps: 
+      - name: 'Checkout'
+        uses: actions/checkout@v3
+        with:
+          ref: 'main'
+          persist-credentials: false
+
+      - name: 'Setup Node v20'
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: 'Setup git user'
+        run: |
+          git config user.name "${{ github.actor }}"
+          git config user.email "bot@callstack.io"
```

Comment:
@cortinico let me know under what e-mail address `react-native-community-bot` will be :pray:

Reply:
We've been using `bot@reactnative.dev` for commit's sake, but we don't have access to that email FYI

Reply:
alright, let's use it 👍